### ISSUE TITLE
Color contrast ratio tool for where colors are specified

### ISSUE THREAD

**Issue by k7hpn** [MCLD/greatreadingadventure#1291]
There are one or more places where colors can be selected by staff members in Mission Control (e.g. categories (accessed via `/MissionControl/Categories`)). Anywhere that a staff member is able to change color foreground/background for text we should have a tool that can determine if the contrast meets the 4.5:1 ratio specified in the [WCAG 2.1 AA](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html) specification and flag or disallow colors which do not meet this contrast ratio.

**Comment by k7hpn** [comment]
Fixed in #1293.
